README.md

```diff
@@ -112,16 +112,22 @@ and using the scoverage scalac plugin (`compileScoverageScala`).
 
 In cases where you only wish to generate reports / validate coverage, but are not interested in publishing the code,
 it is possible to only compile the code with the scoverage scalac plugin, thus reducing build times significantly.
-In order to do so, simply add the arguments `-x compileScala` to the gradle execution.
-For example: `gradle reportScoverage -x compileScala`.
+In order to do so, simply add the arguments `-PscoverageCompileOnly` to the gradle execution.
+For example: `gradle reportScoverage -PscoverageCompileOnly`.
 
+Note that this mode is incompatible with parallel builds in multi-module projects.
 
 ### Compatibility with Consistent Versions Plugin
 
 In order for the plugin to work alongside [Palantir's consistent versions plugin](https://github.com/palantir/gradle-consistent-versions),
 the Scala version must be manually configured (via `scoverageScalaVersion`); otherwise, the plugin will attempt to
 resolve the compilation classpath, which is prohibited by the versions plugin.
 
+Migration to 6.1.1
```

Comment:
I think this is what would normally be considered a breaking change - should we be releasing 7.0.0 here?

I'm sure there is a better way of managing updates to the README, but for now could we remove this file from the PR?

Reply:
@maiflai I was counting on you to make the right decision on whether this should bump the major version or not :)